src/main/java/frc/robot/subsystems/GBSubsystem.java

```diff
@@ -12,11 +11,6 @@ public GBSubsystem(String logPath) {
 		this.logPath = logPath;
 	}
 
-	private String getCurrentCommandName() {
-		Command currentCommand = getCurrentCommand();
-		return currentCommand != null ? currentCommand.getName() : "no command is currently running on the subsystem";
-	}
-
```

Comment:
make a function that takes a command and automatically updates the current command name variable.
additionally the variable needs to be a private variable in GBSubsystem and should have this function returning it.